gateway/1.5/modules/ROOT/pages/policies-pdk-template-caching-policy.adoc

```diff
@@ -0,0 +1,424 @@
+= Data Caching Policy Template
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Use the data caching policy template as an example of how to xref:policies-pdk-configure-features.adoc#share-data-between-workers-and-caching.adoc[configure policy caching].
+
+== Before You Begin
+
+. xref:policies-pdk-prerequisites.adoc[]
+
+== Policy Use Case
+
+An antique store that buys and sells items needs to track the stock of each item in its collection and who buys it. To track the inventory movement, the company uses an API where each unique path represents a different item.. 
```

Comment:
```suggestion
An antique store that buys and sells items needs to track the stock of each item in its collection and who buys it. To track the inventory movement, the company uses an API where each unique path represents a different item.
```

Reply:
I suggest you use either collection and catalog to be consistent. 
Same with inventory and stock